mitodl/hugo-course-publisher
# anchor shortcode and updated markdown

#### What are the relevant tickets?
Closes https://github.com/mitodl/hugo-course-publisher/issues/167

#### What's this PR do?
Adds anchor tag shortcode and updated markdown.

#### How should this be manually tested?
Run the site, verify that the links in the issue work either in your local instance or on the deploy preview.
